server/text_generation_server/models/__init__.py

```diff
@@ -294,7 +304,7 @@ class ModelType(enum.Enum):
     __GLOBALS[data.name] = data.value["type"]
 
 
-def get_model(
+def _get_model(
```

Comment:
I think this `_get_model` and `get_model` naming is not really descriptive. Maybe this should still be named `get_method` and the function below `get_model_with_lora_adapters` or something like that?

Reply:
updated to use `get_model_with_lora_adapters` and renamed `_get_model` back to `get_model`